src/main/java/marquez/api/CreateJobRunRequest.java

```diff
@@ -0,0 +1,27 @@
+package marquez.api;
+
+import com.fasterxml.jackson.annotation.JsonCreator;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import java.util.UUID;
+
+public final class CreateJobRunRequest {
+  private final UUID jobRunDefinitionGuid;
+  private final JobRunState.State currentState;
+
+  @JsonCreator
+  public CreateJobRunRequest(
+      @JsonProperty("job_run_definition_guid") final UUID jobRunDefinitionGuid) {
+    this.jobRunDefinitionGuid = jobRunDefinitionGuid;
+    this.currentState = JobRunState.State.NEW;
```

Comment:
TODO(code review): push this down to the service layer, since state isn't part of the request.